### ISSUE TITLE
I tried your autoaugment policies to my image-classification learning(imagenet-pretrained fine tuning).

### ISSUE THREAD

**Issue by bemoregt** [DeepVoltaire/AutoAugment#13]
Hi, @DeepVoltaire 

I tried your autoaugment policies to my image-classification learning(imagenet-pretrained fine tuning).

But, accuracy is not good than default augmentation.

What's wrong to me?

Autoaugmentation is not suitable for transfer learning or finetuning?

Thanks at any rate.

**Comment by Jongchan** [comment]
@bemoregt 
It seems that you are fine-tuning an ImageNet-pretrained model on ImageNet dataset. But AutoAugment is never validated on transfer learning situations. Only training from scratch.
I've done [training from scratch](https://github.com/DeepVoltaire/AutoAugment/issues/7), and this repository's implementation indeed outperforms the baseline if trained from scratch.

Possibly, you need longer epochs to train, or just use AutoAugment when you train from scratch.
